src/vs/workbench/services/telemetry/electron-browser/telemetryService.ts

```diff
@@ -33,7 +34,10 @@ export class TelemetryService extends Disposable implements ITelemetryService {
 	) {
 		super();
 
-		if (!environmentService.isExtensionDevelopment && !environmentService.args['disable-telemetry'] && !!productService.enableTelemetry) {
+		if (environmentService.args['perf-test']) {
```

Comment:
There are other environment variables that vscode is using to capture different types of perf. https://github.com/microsoft/azuredatastudio/blob/master/src/vs/platform/environment/node/argv.ts#L58 Maybe we need investigation to see the new way they are capturing perf and use that?

Reply:
Thanks Anthony, let me take a look at this. Would be great to leverage something else instead of using our own home grown service if we don't have to.